src/esp/gfx/MeshVisualizerDrawable.cpp

```diff
@@ -21,6 +21,9 @@ MeshVisualizerDrawable::MeshVisualizerDrawable(
 
 void MeshVisualizerDrawable::draw(const Magnum::Matrix4& transformationMatrix,
                                   Magnum::SceneGraph::Camera3D& camera) {
+  // Check that getMesh() doesn't error, i.e. that a mesh exists.
+  getMesh();
```

Comment:
Just put it here for the record:
You should not call this function here simply because it contains a sanity check within.
Instead, you can create a function called e.g., `glMeshExists()` which would tell you if the mesh is nullptr or not.